### PR TITLE
Multi-select email subscription topics

### PR DESCRIPTION
**Pull request by aaronschachter**
#### What's this PR do?

Allows an admin to subscribe users to multiple topics when uploading. Also fixes a typo in the help text 👨‍🎓 

#### How should this be reviewed?
👀 

#### Relevant tickets

This was a nice-to-have reported by @ngjo, let it be had!

### CODE REVIEW

File: app/Jobs/ImportEmailSubscription.php [hunk 1]
```diff
@@ -108,9 +108,11 @@ private function createUser()
     private function updateUser($user)
     {
         $existingTopics = ! empty($user->email_subscription_topics) ? $user->email_subscription_topics : [];
+        $newTopics = array_unique(array_merge($existingTopics, $this->email_subscription_topics));
```
Comment (aaronschachter) [inline app/Jobs/ImportEmailSubscription.php]: @katiecrane I was able to add duplicate subscription topics, this seems like it'd be a useful sanity check to add into Northstar validation?

Comment (katiecrane) [reply app/Jobs/ImportEmailSubscription.php]: @aaronschachter that is a great call! right now Northstar assumes it is getting the complete list of subscription topics for a user with no duplicates.

Comment (katiecrane) [reply app/Jobs/ImportEmailSubscription.php]: also wow `array_unique`, didn't realize that existed, now there's some other code I can simplify 😈

Comment (katiecrane) [reply app/Jobs/ImportEmailSubscription.php]: I made a barebones card for this Northstar issue so we don't forget about it! https://www.pivotaltracker.com/story/show/165074676

**Pull request merged**